source/language/subroutines.rst

```diff
@@ -2,13 +2,13 @@ Subroutines
 ===========
 
 Subroutines are declared using the statement ``def name(parameters) -> output_type { body }``.
-Zero or more quantum bits
-and classical values are passed to the subroutine by reference or name in ``qargs``.
-Classical types are passed by value in ``parameters``.
+The subroutine will define zero or more parameters as input, consisting of both quantum and classical arguments.
+Quantum bits and registers are passed in by reference or name, while classical types are passed in by value.
```

Comment:
These are pre-existing problems, but since we're touching it right now:

- I'm not sure what "passed in by ... name" means entirely in this context.
- The passing type of classical values is a bit more complex than this - arrays are passed by reference.  We might want to just leave it as-is for the time being, though, since the types and casting WG will propose a PR to add a "by reference" modifier for subroutine arguments.